### ISSUE TITLE
timebox time limit input issue

### ISSUE THREAD

**Issue by dae** [ankitects/anki#1617]
https://forums.ankiweb.net/t/its-possible-to-insert-a-comma-in-a-place-where-it-shouldnt-be-possible/16884

**Comment by danishprakash** [comment]
Seems like fixing/doing input validation, mind If I take a shot at this?

**Comment by Spooghetti420** [comment]
I believe I've found out the cause of the bug: whenever the maximum size is reduced to <1000, the bug no longer appears, whereas once it goes over that threshold, you can suddenly type in a comma again. There isn't apparently any difference between the QSpinBox which tracks the timebox time limit and the one that tracks the learn ahead limit, so what gives?

Well, when the number is over 1,000, it's sometimes conventional to write it with a comma to make it easier to read. This wouldn't be possible with numbers below 1,000, but it is once they go over that. It seems Qt knows this, which is why it lets you type the number with a comma, since people might naturally want to write larger numbers that way. 
Basically, I don't really think this is a bug, and even if it were, I don't think it can be "fixed" on our part. Plus, as the poster on the forums said, it doesn't affect Anki's settings, so this should be a purely visual phenomenon.

**Comment by dae** [comment]
@danishprakash sorry for not getting back to you in time

@Spooghetti420 thanks for looking into it! I guess it is Qt that is allowing a comma when the maximum input is larger than 999. It seems the comma is stripped even when using a language that treats a comma as a decimal separator, so it's not entirely consistent, but probably not a big deal.
